Nitrox.Subnautica.Assets/LanguageFiles/Romanian.json

```diff
@@ -0,0 +1,37 @@
+{
+	// Main menu
+	"Nitrox_Multiplayer": "Multiplayer",
+	"Nitrox_AddServer": "Adaugă server",
+	"Nitrox_ConnectTo": "Conectează la",
+	"Nitrox_AddServerName": "Nume:",
+	"Nitrox_AddServerHost": "Gazdă:",
+	"Nitrox_AddServerPort": "Port:",
+	"Nitrox_AddServerAdd": "Adăugă",
+	"Nitrox_JoinServerIpAddress": "Adresa IP a serverului",
+	"Nitrox_JoinServerPassword": "Parola:",
+	"Nitrox_JoinServerFailed": "Nu te poți alătura serverului",
+	"Nitrox_Cancel": "Anulare",
+	"Nitrox_Join": "Alătură-te",
+	"Nitrox_PlayerColor": "Culoarea jucătorului",
+	"Nitrox_EnterName": "Introdu numele de jucător",
+	"Nitrox_SubmitPassword": "Trimite parola",
+	"Nitrox_MultiplayerLoaded": "Clientul Multiplayer încărcat ...",
+	
```

Comment:
Minor: remove spaces between as to keep consistency with other language files

Reply:
Fixed